packages/salesforcedx-vscode-lwc-next/package.json

```diff
@@ -25,6 +25,7 @@
   "dependencies": {
     "@salesforce/salesforcedx-utils-vscode": "44.8.0",
     "ajv": "^6.1.1",
+    "eslint-plugin-lwc": "^0.5.1",
     "lwc-language-server": "1.6.7",
```

Comment:
This also needs to be updated, @rsalvador - would ^1.7.0 work or are you already higher on the version number for 218?

Reply:
the latest is 1.7.0 for 218